### PR TITLE
Compatibility between Python 2 and 3

### PR DESCRIPTION
**Pull request by pWhitS**
These are some minor updates for compatibility between Python2 and Python3. 

Example changes:
- import \_\_future__ modules
- it.next() -> next(it)
- list(items)

### CODE REVIEW

File: splunklib/ordereddict.py [hunk 1]
```diff
@@ -71,9 +76,9 @@ def popitem(self, last=True):
         if not self:
             raise KeyError('dictionary is empty')
         if last:
-            key = reversed(self).next()
+            key = next(reversed(self))
         else:
-            key = iter(self).next()
+            key = next((self).next())
```
[the remark this answers is not in the document]
Comment (pWhitS) [reply splunklib/ordereddict.py]: Good catch... definitely an oversight on my part. I pushed a change to remove the .next()

**Comment by pWhitS** [comment]
Closed and reopened to rerun the build tests. Different tests are failing on each run. I think the failing checks may not related to this change.

**Comment by bparmar-splunk** [comment]
Hi @pWhitS,
Could you please re-push again with all relevant changes which includes resolved conflicts?

Also, when changes are pushed, it will trigger the Github CI workflow to execute all test cases with respect to this change. It would help us to troubleshoot if any of the test case fails.

**Comment by ashah-splunk** [comment]
Closing the PR as no response received. @pWhitS current SDK supports both Python 2 and Python 3 versions.

**Pull request closed without merging**